Assets/Scripts/R&D Tree/RDTreeManager.cs

```diff
@@ -0,0 +1,60 @@
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class RDTreeManager : MonoBehaviour
+{
+    [SerializeField] private GameObject _skillTree;
+
+    private RDControls _controls;
+
+    public static RDTreeManager Instance;
```

Comment:
Singleton instance is not needed with the gamemanager. For the demo, you can just reference to a GameObject with the instance